extensions/eventtap/internal.m

```diff
@@ -173,12 +172,18 @@ static int eventtap_start(lua_State* L) {
                               eventtap_callback,
                               e);
 
-    CGEventTapEnable(e->tap, true);
-    e->runloopsrc = CFMachPortCreateRunLoopSource(NULL, e->tap, 0);
-    CFRunLoopAddSource(CFRunLoopGetMain(), e->runloopsrc, kCFRunLoopCommonModes);
+    if (e->tap) {
+        e->running = true;
 
-    lua_settop(L,1);
-    return 1;
+        CGEventTapEnable(e->tap, true);
+        e->runloopsrc = CFMachPortCreateRunLoopSource(NULL, e->tap, 0);
+        CFRunLoopAddSource(CFRunLoopGetMain(), e->runloopsrc, kCFRunLoopCommonModes);
+
+        lua_settop(L,1);
+        return 1;
+    } else {
+        showError(L, "Unable to create eventtap.  Is Accessibility enabled?");
+    }
```

Comment:
This is missing a return statement. We should probably have the lua_settop(L, 1); return; here. This method is supposed to return the eventtap object.
We could also arguably return nil and document that that happens in the event of an error.